src/test/scala/beam/cosim/helics/BeamFederateSpec.scala

```diff
@@ -53,8 +54,14 @@ class BeamFederateSpec extends FlatSpec with Matchers with BeamHelper with Befor
     } yield (f1Result, f2Result)
     try {
       Await.result(aggregatedFuture, 5.minutes)
-      require(chargingPlugInEvents.get() > 0)
-      require(chargingPlugOutEvents.get() > 0)
+      assert(
+        chargingPlugInEvents.get() > 0,
+        s"Charging Plugin Event expected to be greater than 0 but is ${chargingPlugInEvents.get}"
+      )
+      assert(
+        chargingPlugOutEvents.get() > 0,
+        s"Chaging Plugout Event expected to be greater than 0 but is ${chargingPlugOutEvents.get}"
+      )
```

Comment:
I believe the lines `57-64` could be simplified with Matchers:
```scala
chargingPlugInEvents.get() should be > 0
chargingPlugOutEvents.get() should be > 0
```
This way description is not needed

Reply:
fair enough - then the line number should cover which event